docs/src/Tabs.doc.js

```diff
@@ -55,33 +62,53 @@ card(
 card(
   <Example
     name="Example"
-    direction="row"
     defaultCode={`
 function TabExample() {
   const [activeIndex, setActiveIndex] = React.useState(0);
+  const [wrap, setWrap] = React.useState(false);
   const handleChange = ({ activeTabIndex, event }) => {
     event.preventDefault();
     setActiveIndex(activeTabIndex)
-  }
+  };
+  const TABS = [
+    { href: "https://pinterest.com", text: "Boards for You" },
+    { href: "https://pinterest.com", text: "Pins for You" },
+    { href: "https://pinterest.com", text: "❤" },
+    { href: "https://pinterest.com", text: "Following" },
+    { href: "https://pinterest.com", text: "People to Follow" },
+  ];
+  const sizes = [
+    ['md', 0], ['md', 1],
+    ['lg', 0], ['lg', 1],
+  ];
   return (
-    <Tabs
-      tabs={[
-        {
-          text: "Boards",
-          href: "#"
-        },
-        {
-          text: "Pins",
-          href: "#"
-        },
-        {
-          text: "Topics",
-          href: "#"
-        }
-      ]}
-      activeTabIndex={activeIndex}
-      onChange={handleChange}
-    />
+    <Stack gap={2}>
+      <Row gap={2} padding={2}>
+        <Label htmlFor="wrap">
+          <Text>Wrap</Text>
+        </Label>
+        <Switch
+          id="wrap"
+          onChange={() => setWrap(!wrap)}
+          switched={wrap}
+        />
+      </Row>
+      {sizes.map(([size, gap]) => (
+        <Box key={size + String(gap)}>
+          <Heading size="sm">size={size}, gap={gap}</Heading>
+          <Box maxWidth={500} overflow="auto" borderSize="sm" padding={1}>
```

Comment:
@jackhsu978 I do like this solution to no longer have the focus state overlap, add a padding of `1` (4px) to the container.

Reply:
@christianvuerings My previous iteration did prevent the focus state overlap (adding 0.5 boint padding, so that the gap between tabs is 1 boint, exactly the same as the width of the blue a11y box shadow), but I took it out because you commented that we shouldn't be adding the padding to each tab component. Do you want me to add that back? It'd be effectively adding `gap` 0.5 boint to the Row component (except that I have to do dangerouslySetInlineStyle since the `gap` prop must be an integer)

@christianvuerings @ashleyseto which one of the following 3 is preferred?
![Screenshot 2020-06-30 11 13 39](https://user-images.githubusercontent.com/5341184/86162176-3fccd900-bac3-11ea-940b-9aad8dac3762.png)

Reply:
ok, based on Annie's feedback, we want the gap=0 variant, despite the blue accessibility outline overlapping the adjacent tab.